themes/godotengine/layouts/more.htm

```diff
@@ -17,6 +17,9 @@ <h1 class="intro-title">{% placeholder title %}</h1>
       <a href="/contact" class="title-font {% if this.page.id == 'contact' %} active {% endif %}">
         Contact
       </a>
+      <a href="/privacy-policy" class="title-font {% if this.page.id == 'privacy-policy' %} active {% endif %}">
+        Privacy
```

Comment:
I had to shorten it as it made the `main` div on the left too small. We should probably reduce the horizontal space between each navbar item to work it around, or redesign the thing.